skyportal/schema.py

```diff
@@ -171,10 +171,10 @@ class PhotBaseFlexible(object):
                           required=True)
 
     instrument_id = fields.Field(description='ID of the `Instrument`(s) with which the '
-                                 'photometry was acquired. '
-                                 'Can be given as a scalar or a 1D list. '
-                                 'If a scalar, will be broadcast to all values '
-                                 'given as lists. Null values are not allowed.',
+                                      'photometry was acquired. '
```

Comment:
Next time I edit this file, upon save, emacs will automatically change this back.... ;)  War of the auto-formatters!

Reply:
Fixed

Reply:
Oh that wasn't a request/complaint, just remarking that it's funny that this happens ;)